packages/admin/admin/src/common/toolbar/Toolbar.tsx

```diff
@@ -24,21 +23,17 @@ export interface ToolbarProps
     hideTopBar?: boolean;
 }
 
-type OwnerState = {
-    headerHeight: number;
-};
-
-const Root = createComponentSlot(Paper)<ToolbarClassKey, OwnerState>({
+const Root = createComponentSlot(Paper)<ToolbarClassKey>({
     componentName: "Toolbar",
     slotName: "root",
 })(
-    ({ ownerState }) => css`
+    css`
         position: sticky;
         z-index: 10;
         display: flex;
         flex-direction: column;
         justify-content: center;
-        top: ${ownerState.headerHeight}px;
+        top: 0;
```

Comment:
Isn't removing this a breaking change for all toolbar usages outside of data grid?

Reply:
@jamesricky and me looked into this thing. We think that mui introduced an position relative somewhere, and therefor setting the top position is not necessary anymore.

Reply:
But what if this behavior was required for the toolbar in non-data-grid-scenarios? For instance, here: https://deploy-preview-2837.storybook.comet-dxp.com/?path=/story/docs-components-master--master

<img width="2262" alt="toolbar space" src="https://github.com/user-attachments/assets/d57c53c6-ee65-4fc2-998b-1e6e28275555">

Reply:
Nice catch, @jamesricky, and I was not aware of this. I reverted the removal of headerHeight and made the `headerHeight` overridable on the ToolBar.